### ISSUE TITLE
Export to Excel

### ISSUE THREAD

**Issue by SharmaAntriksh** [DaxStudio/DaxStudio#843]
There should be a feature to extract data directly to Excel :)

**Comment by dgosbell** [comment]
So you can already output data to Excel. If you choose the Static Excel option from the Output dropdown it will send the results of your query to an xlsx file.

**Comment by SharmaAntriksh** [comment]
Agreed, however that is only limited to one query/file at a time, what I meant is the ability to extract every table into either a separate excel files or just one file. Like how Bravo does.

**Comment by dgosbell** [comment]
That would probably not be too hard to add, but the main issue is that Excel has a limit of 1 million rows per sheet so it would really only work for small models. That's why we did CSV / SQL first since they are only really limited by hard disk space.

**Comment by SharmaAntriksh** [comment]
Yup, totally forgot about Excel's limit for a moment there!
